### PR TITLE
wasm: Don't include all image codecs on the lsp

### PR DESCRIPTION
**Pull request by ogoffart**
Saves 1M in the binary

We use the browser to render the images anyway

Note: i-slint-core still bring image with png and jpeg codecs, as well
as svg deps. We just need the image-decoders feature to load the builtin
images, of these there are only svg.

(also has a cleanup stuff for the interpreter Cargo.toml)

### CODE REVIEW

File: internal/renderers/femtovg/Cargo.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ once_cell = "1.5"
 pin-weak = "1"
 scoped-tls-hkt = "0.1"
 raw-window-handle = { version = "0.5", features = ["alloc"] }
-femtovg = { version = "0.7.0" }
+femtovg = { version = "0.7.0", default-features = false }
```
Comment (tronical) [inline internal/renderers/femtovg/Cargo.toml:36]: This doesn't seem quite right to me. I believe the code here needs the image-loading feature of femtovg.

Comment (ogoffart) [reply internal/renderers/femtovg/Cargo.toml:36]: Why do we need that? The image are loaded by i-slint-core.
if we were using this feature, i think we'd have a compilation error.

Comment (tronical) [reply internal/renderers/femtovg/Cargo.toml:36]: Yes, for our images that's true. But the image dependency in femtovg also enables it to load pngs from ttf files, for color emojis:

https://github.com/femtovg/femtovg/blob/46fdb65a569d75639f67d995b11e6fb7ea30003a/src/text/font.rs#L195

Comment (ogoffart) [reply internal/renderers/femtovg/Cargo.toml]: Ah ok, i didn't know that.
I removed this part then.
Thanks for noticing.

**Review by tronical — APPROVED** [review]

**Pull request merged**